styles.css

```diff
@@ -0,0 +1,183 @@
+a {
+  text-decoration: none;
+}
+
+.header {
+  width: 375px;
+  height: 92px;
+  background-color: #303030;
+  display: flex;
+  justify-content: space-between;
+}
+
+.name {
+  color: white;
+  height: 24px;
+  font-family: Roboto, serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: center;
+  position: relative;
+  top: 20px;
+  left: 10px;
+}
+
+.name p {
+  height: 24px;
+  width: 187px;
+}
+
+.burger {
+  height: 24px;
+  width: 20px;
+  margin-left: 40px;
+  margin-right: 5px;
+  margin-bottom: 15px;
+  border-radius: 0;
+  align-self: center;
+  position: relative;
+  top: 10px;
+  right: 10px;
+}
+
+.burger a {
+  background-color: white;
+  margin-bottom: 5px;
+  padding-bottom: 0;
+}
+
+.main {
+  border-top: 1px solid slateblue;
+  color: white;
+  background-color: #151515;
+  width: 375px;
+  height: 610px;
+}
+
+.title {
+  margin-left: 20px;
+}
+
+.head {
+  margin-left: 20px;
+}
+
+.content {
+  margin-left: 20px;
+}
+
+h1 {
+  border-left: 20px;
+  color: #ff6b00;
+  height: 124px;
+  width: 327px;
+  left: 24px;
+  top: 120px;
+  font-family: Crete Round, serif;
```

Comment:
- [ ] Nice work you have done so far 👍, however, all the fonts you used in your CSS are not working because the fonts are not linked to your work currently. Please try checking my comment on how to link Google font to your `HTML` or `CSS` so that your fonts will be active.

Reply:
Done.